### ISSUE TITLE
Formatting wrap text will result in infinite indentation and auto-add double quotes for value

### ISSUE THREAD

**Issue by aron19951209** [mblode/vscode-twig-language#27]
### Before formatting
<img width="658" alt="螢幕快照 2019-04-13 上午1 37 05" src="https://user-images.githubusercontent.com/33840671/56055662-a796e700-5d8c-11e9-813d-9fdc08327205.png">

### After formatting
<img width="657" alt="螢幕快照 2019-04-13 上午1 37 21" src="https://user-images.githubusercontent.com/33840671/56055674-b087b880-5d8c-11e9-9555-5fd5b8a096f6.png">

Two problems:
1. The wrap text would indent Infinitely.
2. Shouldn't add double quotes for value automatically. `class=p`
( This should be a personal habit. )

The following code give you to test:
```html
<div class="p">
    <h4>Market — your business
        {{ i + 1 }}</h4>
    <p class="mt-sm">
        Take the guesswork out of marketing with 
        built-in tools that help you create, execute,
        and analyze campaigns on Facebook and Google.
    </p>
</div>
```

**Comment by mblode** [comment]
You could set the wrap to 40 characters and have no issues. Otherwise, the formatter doesn't respond well to enter keys/line breaks in paragraphs of text

**Comment by mblode** [comment]
Fixed repeated paragraph indentation in 0.10.0. Tests check that a second format makes no changes.
